### PR TITLE
Added const to argv type for better interoperability.

### PR DESCRIPTION
**Pull request by jgonzalezdr**
This change is [Reviewable](https://reviewable.io/reviews/jarro2783/cxxopts/98)

### CODE REVIEW

**Comment by jarro2783** [comment]
This doesn't pass the tests. `argv` actually needs to be changed.

**Comment by jgonzalezdr** [comment]
@jarro2783: Compilation is now fixed. BTW, I'll also create a PR for the master branch.

File: test/catch.hpp [hunk 1]
```diff
@@ -10280,7 +10280,7 @@ namespace Catch {
 #ifndef __OBJC__
 
 // Standard C/C++ main entry point
-int main (int argc, char * argv[]) {
+int main (int argc, const char * argv[]) {
```
Comment (jarro2783) [inline test/catch.hpp]: This whole file is copied verbatim from the `catch` distribution and so is likely to be overwritten. You can probably leave this one out.

Comment (jgonzalezdr) [reply test/catch.hpp]: You're right, I went a bit too happy with Find+Replace!

**Comment by jarro2783** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1, 2 of 3 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/jarro2783/cxxopts/98#-:-L76QYUB7dXGc2cxNfpJ:bnfp4nl)*

**Pull request merged**